src/util.cpp

```diff
@@ -30,6 +36,20 @@ namespace hypha
         return std::move(cgs);
     }
 
+    ContentGroups getDAOContent(const eosio::name &dao_name)
```

Comment:
This is how the DAO document is constructed, translating the dao_name -> ContentGroups object -> hash, then read the document from storage.  This feels a bit heavy on CPU.  We can skip this processing by having actions that accept the ```dho``` document's hash as an input instead of the ```dao_name```.  

However, this isn't as readable when looking through the blockchain action logs and block explorers because we'd have to memorize the DHO hashes. One way to support both is adding an indexed cache table in the contract that stores a lookup of ```dao_name``` to hash so it's easier on CPU.  

(We have been OK on CPU thus far I believe, but I don't know how close we are to the threshold overall.)